src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -0,0 +1,511 @@
+                {
+                  href: Nav.getLink('groups'),
+                  ...newTabLinkProps,
+                },
+                ['groups']
+              ),
+              ' to access this workspace.',
+            ]),
+            ..._.map(
+              ({ membersGroupName }) => div({ style: { margin: '0.5rem', fontWeight: 500 } }, [membersGroupName]),
+              authorizationDomain
+            ),
+          ]
+        ),
+      h(
+        RightBoxSection,
+        {
+          title: 'Tags',
+          info: span({}, [
+            (busy || !tagsList) && tagsPanelOpen && spinner({ size: 1, style: { marginLeft: '0.5rem' } }),
+          ]),
+          initialOpenState: tagsPanelOpen,
+          onClick: () => setTagsPanelOpen(!tagsPanelOpen),
+        },
+        [
+          div({ style: { margin: '0.5rem' } }, [
+            div({ style: { marginBottom: '0.5rem', fontSize: 12 } }, [
+              `${brand.name} is not intended to host personally identifiable information.`,
+              h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
+                `${brand.name} is not intended to host personally identifiable information. Do not use any patient identifier including name,
+              social security number, or medical record number.`,
+              ]),
+            ]),
+            canEdit &&
+              div({ style: { marginBottom: '0.5rem' } }, [
+                h(WorkspaceTagSelect, {
+                  menuShouldScrollIntoView: false,
+                  value: null,
+                  placeholder: 'Add a tag',
+                  'aria-label': 'Add a tag',
+                  onChange: ({ value }) => addTag(value),
+                }),
+              ]),
+            div({ style: { display: 'flex', flexWrap: 'wrap', minHeight: '1.5rem' } }, [
+              _.map((tag) => {
+                return span({ key: tag, style: styles.tag }, [
+                  tag,
+                  canEdit &&
+                    h(
+                      Link,
+                      {
+                        tooltip: 'Remove tag',
+                        disabled: busy,
+                        onClick: () => deleteTag(tag),
+                        style: { marginLeft: '0.25rem', verticalAlign: 'middle', display: 'inline-block' },
+                      },
+                      [icon('times', { size: 14 })]
+                    ),
+                ]);
+              }, tagsList),
+              !!tagsList && _.isEmpty(tagsList) && i(['No tags yet']),
+            ]),
+          ]),
+        ]
+      ),
+      h(
+        RightBoxSection,
+        {
+          title: 'Notifications',
+          initialOpenState: notificationsPanelOpen,
+          onClick: () => setNotificationsPanelOpen(!notificationsPanelOpen),
+        },
+        [h(WorkspaceNotifications, { workspace })]
+      ),
+    ]),
+  ]);
+};
+
+const WorkspaceDashboard: (props: WorkspaceDashboardProps) => typeof WorkspaceDashboardComponent = _.flow(
```

Comment:
`typeof WorkspaceDashboardComponent` is incorrect here. `forwardRef` takes a forward ref render function and returns a component. The second argument is different (ref vs context).